### ISSUE TITLE
Air Quality Lesson 2 Lit Review

### ISSUE THREAD

**Issue by jfmartinez4** [ciesin-geospatial/TOPSTSCHOOL-air-quality#17]
Task: Review Module 2 Lesson 2,  "Exploring Air Quality, Social Vulnerability, and Health Outcomes in Metro Detroit." Read through the lesson, conduct search and add relevant citations that support text already written. If research shows more accurate info that is not written in the google document, then rewrite google document to reflect research **in suggestion mode.**

Done is: Add changes and in-text citations and bibliography to the module 2 google review document **in suggestion mode.**

Google review document: https://docs.google.com/document/d/1ftOew6_FaUwINDpGmz-HH9UrfuYERVu4wqAHfwD766w/edit?tab=t.0#heading=h.2wmz5hf3ff45

module 2 Lesson 2 GitHub Page: https://ciesin-geospatial.github.io/TOPSTSCHOOL-air-quality/m202-svi-tri-icis-places.html

Module 2 Air Quality citations: https://www.zotero.org/groups/5638572/topstschool/collections/IC4J25VH

**Comment by jfmartinez4** [comment]
@Bravo-Juan I added you to this task. please review and let me know if you have any questions. I've set a deadline for updates for next week November 1.

**Comment by jfmartinez4** [comment]
Finished research and looking for 18 citations. 

- Next steps: Send list of citations in a bibliography format on google doc. Look over them and see which ones inform the module. 
- Added citations on Zotero. 
-

**Comment by Bravo-Juan** [comment]
@jfmartinez4 Hello Juan, I have just added the citations that I found to be relevant for Module 2 Lesson 2 to the google document. I also wanted to mention that from the 18 citations, one of them was quite outdated, so I left that one out. 

You can find my citations at the end of Lesson 2 in the Google document. I made a little section in between the "Key Learning Points" section and the "Lesson 3" section. 

Please let me know once they have been verified and what I should do next, I look forward to your next response.

**Comment by jfmartinez4** [comment]
@Bravo-Juan At a quick glance, these citations look great. I couldn't find your in-text citations, I only saw the bibliography. Please add the in-text citations in the appropriate places of the lesson narrative. Let me know if you have any questions.

**Comment by Bravo-Juan** [comment]
@jfmartinez4 Hello Juan, yea I still haven't added the in-text citations or made any updates on the text because you mentioned that you first wanted to see the list of citations that I found to see if they were relevant.

Would you like for me to begin to look into my citations and start adding the in-text citations for the appropriate places and start seeing if there's more up-to-date information in my citations so that I can add it to the narrative?

**Comment by jfmartinez4** [comment]
@Bravo-Juan Yes, I also added some comments to the citations.

**Comment by jfmartinez4** [comment]
@Bravo-Juan I added comments in the google doc. Go ahead adding the in-text citations

**Comment by jfmartinez4** [comment]
@Bravo-Juan Please continue to add hyperlinks and citations to the document. Please reply when finished.;

**Comment by Bravo-Juan** [comment]
@jfmartinez4 Hello Juan, I just finished transferring over the possible references/hyperlinks for the remainder of the lesson, starting from page 85. I also added some references on some sections prior to page 85 that were missed by Linda. Please let me know what I need to do next.

I also had a question regarding the list of citations that was inserted into the lesson at the beginning of the task when you asked me to share the references I found. What would you like me to do with this list? Shall I delete it or only delete the bibliographies that were not mentioned?

If you'd like I could also modify this list to include all the references that were mentioned in the lesson by me and Linda. However, before I am able to do that, I think we would need confirmation on the exact references that will be used to cite each suggested section.

**Comment by Bravo-Juan** [comment]
Actually, I just realized that this list of references is probably created on its own through the quarto file, so me listing them out wouldn't really have any effect. If this is the case, would you like for me to just erase the list I had made in the google document?

**Comment by jfmartinez4** [comment]
@Bravo-Juan I usually grab the DOI from the comment. If there is no DOI, then you can write a comment with the full citation. You don't have to erase the list, just add a comment to it mentioning what it is.

**Comment by Bravo-Juan** [comment]
@jfmartinez4 All of the references that I suggested throughout the document should have both the in-text citation and the full reference. However, do let me know if you'd like me to get rid of the in-text citation or full reference and just leave the DOI in the comment to make things easier to read.

I also did what you mentioned, I left a comment on the heading of this list to provide some context on what it is.
